src/runtime/hexagon_dma_device_shim.cpp

```diff
@@ -0,0 +1,196 @@
+/*!
+ *  This file is a duplicate of the actual hexagon_dma_device_shim.cpp used to call the DMA driver functions
+ * The definitions in this file a week reference so that these will be called only in case of unavailability of
+ * actual DMA functions.
+ * This file is need only if there is no hexagon SDK support or NO hexagon DMA support, in either csae we replace
+ * the DMA operations with normal memory operations
+ */
+
+#ifndef _DMA_DEVICE_SHIM_C_
+#define _DMA_DEVICE_SHIM_C_
+
+#include "hexagon_mini_dma.h"
+#include "mini_qurt.h"
+#include "hexagon_dma_device_shim.h"
+
+using namespace Halide::Runtime::Internal::Qurt;
+
+typedef struct dma_dummy_lib
+{

```

Comment:
Opening brace should be on same line as typedef (more below...won't call out all of them)